tests/functional/test_all.py

```diff
@@ -252,7 +253,15 @@ def test_execution_plan(self):
 
         result = redis_graph.execution_plan("MATCH (r:Rider)-[:rides]->(t:Team) WHERE t.name = $name RETURN r.name, t.name, $params", {'name': 'Yehuda'})
         expected = "Results\n    Project\n        Conditional Traverse | (t:Team)->(r:Rider)\n            Filter\n                Node By Label Scan | (t:Team)"
-        self.assertEqual(result, expected)
+        self.assertEqual(str(result), expected)
+
+        expected = Operation('Results') \
```

Comment:
Do we need the trailing `\` ?

Reply:
yes

Reply:
Can you explain why ?

Reply:
if I remove it then python not identify that this is a continuation of the code